.github/workflows/ci.yml

```diff
@@ -12,6 +12,44 @@ env:
     TZ: "Europe/Paris"
 
 jobs:
+    phar:
+        name: Compile Phar
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
```

Comment:
Is this really needed ? This will download the whole git history... while only the latest commit is relevant here.

Reply:
thanks, copy/paste from another place, fixed in `master`

Reply:
Cool, thanks :)